examples/logical_enclosures.py

```diff
@@ -34,12 +34,49 @@
     }
 }
 
-
 # Try load config from a file (if there is a config file)
 config = try_load_from_file(config)
 
 oneview_client = OneViewClient(config)
 
+try:
+    # The valid enclosure URIs need be inserted sorted by URI
+    # The number of enclosure URIs is conform enclosure count in enclosure group
```

Comment:
This sentence is not clear to me, I suggest something different:
"The number of enclosure URIs must be in accordance with enclosure count in the enclosure group"
or
"The number of enclosure URIs must be equal to the enclosure count in the enclosure group"